pages/DSNP/DSNP-Archive-Messages.mdx

```diff
@@ -0,0 +1,103 @@
+import { Image } from "@chakra-ui/react";
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Proposed |
+
+## Purpose
+1. Specify the Archivist message format
+
+## Assumptions
+* Chain messages are on Ethereum
+* Message data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378)
+* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use `ecreover`
+to get public keys. A public key also need not be included in a log event for ease of validation.
+* content hashes are created via the same [keccak-256 hashing algorithm](https://en.wikipedia.org/wiki/SHA-3) used by Ethereum.
+
+## Archive Entry
+An archive entry is a combination of data in a DSNP message and from the block in which it is included.
+It is a key-value map consisting of the following fields:
+
+|field|description|type|required?|
+|---|---|---|---|
+| `author` | social identity of author of the content. | bytes | YES
+| `blockHeight` | the block number this message was included in | number | YES
+| `fromAddress`| social identity of message sender | bytes | YES
+| `logIndex` | the index within the logs of this message | number | YES
+| `messageID` | keccak-256 hash of this data, minus the messageID |  bytes3 | YES
+| `dsnpType` | DSNP message type |number/enum. see [DSNP Message Types](/DSNP/DSNP-Message-Types) | YES
+| `dsnpData` | DSNP message data | see fields in [DSNP Messages](/DSNP/DSNP-Messages) | YES
+| `signatures` | list of additional signatures referencing this message | array of Signatures | YES
+| `transactionIndex` | the index of the transaction this message is associated with | number | YES
```

Comment:
I wish there were a nice way to combine our three ordering fields into one, but I don't see a way to do it.